Realm/RLMObject_Private.h

```diff
@@ -75,6 +75,12 @@ FOUNDATION_EXTERN id _Nullable RLMValidatedValueForProperty(id object, NSString
 // Compare two RLObjectBases
 FOUNDATION_EXTERN BOOL RLMObjectBaseAreEqual(RLMObjectBase * _Nullable o1, RLMObjectBase * _Nullable o2);
 
+typedef void (^RLMObjectNotificationCallback)(NSArray<NSString *> *_Nullable propertyNames,
```

Comment:
Initially it was different because I thought `RealmOptional` needed some special handling, and then I didn't bother changing it because I don't think it'd actually meaningfully simply or shorten the code.